documentation-site/pages/components/code.js

```diff
@@ -0,0 +1,65 @@
+import {LiveProvider, LiveEditor, LiveError, LivePreview} from 'react-live';
+// import { useHover } from "./hooks";
+// import {
+//   styled,
+//   withStyle,
+//   withTransform,
+//   createStyled,
+//   useStyletron
+// } from "styletron-react";
+// import { driver, getInitialStyle } from "styletron-standard";
+// import { DESKTOP_BREAKPOINT } from "../const";
+
+// import {Avatar} from '../../../src/avatar/avatar';
+import Avatar from 'baseui/avatar';
```

Comment:
2) Since `Avatar` is undefined, I tried to pass it to `scope`. The import above returned `undefined`. `scope` is working when I pass something simple such as `headerProps` object for the example from the previous commit.

Reply:
It should be 

```js
import {Avatar} from 'baseui/avatar';
```

Reply:
You are right, thanks. I have tried `import {Avatar} from 'baseui/avatar';`as well, has the same results.